datadog_checks_dev/datadog_checks/dev/tooling/commands/release/show/changes.py

```diff
@@ -12,18 +12,19 @@
 
 @click.command(context_settings=CONTEXT_SETTINGS, short_help='Show all the pending PRs for a given check.')
 @click.argument('check', autocompletion=complete_valid_checks, callback=validate_check_arg)
-@click.option('--organization', '-r', default='DataDog')
-@click.option('--tag-pattern', default=None)
-@click.option('--tag-prefix', default=None)
-@click.option('--dry-run', '-n', is_flag=True)
+@click.option('--organization', '-r', default='DataDog', help="The Github organization the repository belongs to")
+@click.option('--tag-pattern', default=None, help="The regex pattern for the format of the tag. Required if the tag doesn't follow semver")
+@click.option('--tag-prefix', default=None, help="Specify the prefix of the tag to use if the tag doesn't follow semver")
+@click.option('--dry-run', '-n', is_flag=True, help="Run the command in dry-run mode")
+@click.option('--since', help="The git ref to use instead of auto-detecting the tag to view changes since")
```

Comment:
```suggestion
@click.option('--since', default=None, help="The git ref to use instead of auto-detecting the tag to view changes since")
```

Reply:
From my testing, if the option isn't specified, the default is already None. Any reason to set it explicitly?

Reply:
Ahh, I see these defaults are on the other options, adding